addon/components/polaris-color-picker/hue-picker.js

```diff
@@ -8,11 +8,7 @@ const VERTICAL_PADDING = 13;
 
 function offsetForHue(hue, sliderHeight, draggerHeight) {
   const slidableArea = sliderHeight - (draggerHeight + VERTICAL_PADDING);
-  return clamp(
-    (hue / 360) * slidableArea + VERTICAL_PADDING,
-    0,
-    sliderHeight - draggerHeight,
-  );
+  return clamp((hue / 360) * slidableArea + VERTICAL_PADDING, 0, sliderHeight - draggerHeight);
```

Comment:
This is another almost-100-char line which should maybe have been left multi-line...